Xamarin.Forms.Core/Editor.cs

```diff
@@ -8,12 +8,9 @@ namespace Xamarin.Forms
 	[RenderWith(typeof(_EditorRenderer))]
 	public class Editor : InputView, IEditorController, IFontElement, ITextElement, IElementConfiguration<Editor>
 	{
-		public static readonly BindableProperty TextProperty = BindableProperty.Create("Text", typeof(string), typeof(Editor), null, BindingMode.TwoWay, propertyChanged: (bindable, oldValue, newValue) =>
-		{
-			var editor = (Editor)bindable;
-			if (editor.TextChanged != null)
-				editor.TextChanged(editor, new TextChangedEventArgs((string)oldValue, (string)newValue));
-		});
+		public static readonly BindableProperty TextProperty = BindableProperty.Create("Text", typeof(string), typeof(Editor), null, BindingMode.TwoWay, propertyChanged: (bindable, oldValue, newValue) 
+			=> (bindable as Editor)?.TextChanged?.Invoke(bindable, new TextChangedEventArgs((string)oldValue, (string)newValue))
```

Comment:
direct cast

Reply:
as casting is faster than prefix casting if not slightly more error prone (null excpetion vs cast exception)